packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1392,6 +1392,17 @@ const schema: SchemaType<DbUser> = {
     hidden: !dialoguesEnabled,
   },
 
+  hideDialogueFacilitation: {
+    type: Boolean,
+    canRead: [userOwns, 'sunshineRegiment', 'admins'],
+    canCreate: ['members'],
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
+    optional: true,
```

Comment:
This should have an explicit `nullable: false`, and a `...schemaDefaultValue(false)`

Reply:
fixed

Reply:
Unfortunately you will then also need to delete the generated migration file, as well as undo any changes to `accepted_schema.sql` and `schema_changelog.json` (you can run `git checkout origin/master -- schema/accepted_schema.sql schema/schema_changelog.json` to overwrite whatever you have with whatever's latest in master).  Then run `yarn makemigrations` again, update the generated migration file, etc, etc.

Reply:
Makes sense. Now did all these things again!